src/main/java/software/sigma/bu003/internship/coparts/entity/Part.java

```diff
@@ -0,0 +1,35 @@
+package software.sigma.bu003.internship.coparts.entity;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.PersistenceCreator;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import javax.validation.constraints.NotNull;
+
+
+@Data
+@Document
+public class Part {
+    @Id
+    private String id;
+    @NotNull
+    @Field("brand")
+    private String brand;
+    @NotNull
+    @Field("code")
+    private String code;
+    @Field("price")
```

Comment:
do you really need @Field if the property name is the same as the field?

Reply:
Deleted